code/src/java/pcgen/io/PCGVer2Parser.java

```diff
@@ -474,18 +469,6 @@ private void cacheLine(String s)
 		cache.put(s.substring(0, s.indexOf(':')), s);
 	}
 
-	private void checkSkillPools()
-	{
-		int skillPoints = 0;
-
-		for (final PCClass pcClass : thePC.getClassSet())
-		{
-			skillPoints += pcClass.getSkillPool(thePC);
```

Comment:
As much as I hate to have to ask this - have you verified this method does not have side effects?  Remember a lot of older code was not written by folks with an object-oriented history, so it doesn't always follow OO rules.